src/platform/assets/components/AssetCard.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div
+    data-component-id="AssetCard"
+    :data-asset-id="asset.id"
+    role="button"
```

Comment:
It might be fine, but I usually suggest using the intended semantic element and then styling it instead of using a different element and declaring the role. There's a lot that can get missed that way.